src/main/sensors/rangefinder.c

```diff
@@ -188,7 +188,7 @@ timeDelta_t rangefinderUpdate(void)
 /**
  * Get the last distance measured by the sonar in centimeters. When the ground is too far away, RANGEFINDER_OUT_OF_RANGE is returned.
  */
-int32_t rangefinderRead(void)
+void rangefinderRead(void)
```

Comment:
We already have `rangefinderUpdate` that does something completely different. So let's call it `rangefinderProcess` instead